build-system/tasks/visual-diff.js

```diff
@@ -116,7 +116,7 @@ function setPercyBranch() {
 async function launchWebServer() {
   const gulpServeAsync = execScriptAsync(
       `gulp serve --host ${HOST} --port ${PORT} ${process.env.WEBSERVER_QUIET}`,
-      {stdio: 'inherit'});
+      getStdioOptions(argv.gulp_debug || argv.webserver_debug));
 
   gulpServeAsync.on('exit', code => {
     if (code != 0) {
```

Comment:
I'm seeing an error code of `null` being returned by `gulp serve` in the `.on('exit', ...)` handler: https://travis-ci.org/ampproject/amphtml/jobs/403246979#L627
```
[18:55:04] ERROR: 'serve' errored with code null. Cannot continue with visual diff tests
```

From https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options, it appears you should be handling errors in `.on('close', ...)` and not `.on('exit', ...)`.